code/datums/critter_mobs/hands.dm

```diff
@@ -21,6 +21,9 @@
 	var/cooldown_overlay = 0
 	var/mob/holder = null
 
+	/// Object for inhand sets the inhands of that item, for limbs with weapons etc
+	var/object_for_inhand = null
```

Comment:
best make it clear this is a type not an instance (because we don't have a type for types)